package.json

```diff
@@ -1,50 +1,45 @@
 {
   "name": "react-page-visibility",
-  "version": "0.0.2",
+  "version": "1.0.0",
   "description": "Declarative, nested, stateful, isomorphic page visibility for React",
   "author": "Gilad Peleg <giladp007@gmail.com> (http://giladpeleg.com)",
   "repository": "pgilad/react-page-visibility",
-  "keywords": "react, components, visibility, page, higher-order, browser",
-  "main": "index.js",
+  "keywords": "react, components, visibility, page, higher-order, browser, react-component",
+  "main": "dist/index.js",
+  "module": "src/index.js",
```

Comment:
Hey @pgilad, so basically we want a transpiled version which hasn't been wrapped by a commonjs module. If you remove this option it means webpack will then import the main (dist/index.js) path which will be imported as a commonjs module, meaning webpack can't tree shake 😱 Please see my original changes in the other PR.